site/docs/getting-started.md

```diff
@@ -327,12 +327,14 @@ arctic.ams.mybatis.ConnectionPassword: {password}                        #MySQL
 arctic.ams.database.type: mysql                                          #系统库类型
 ```
 
+???+note "目前只支持 MySQL 5.x 版本，不支持 MySQL 8"
```

Comment:
our jdbc version do not support mysql 8?

Reply:
yes